doc/ApiDocGeneration/Mashup-sdk-docindex.ps1

```diff
@@ -0,0 +1,27 @@
+[CmdletBinding()]
+Param (
+    [Parameter(Mandatory = $True)]
+    $DocOutHtmlDir,
+    [Parameter(Mandatory = $True)]
+    $ArtifactStagingDirectory,
+    [Parameter(Mandatory = $True)]
+    $ArtifactName
+)
+
+# Copy everything inside of /api out.
+Write-Verbose "Copy index.html and toc.yml out."
```

Comment:
Any reason not to do these step in the Generate-API-Docs.ps1 file directly?

Reply:
It is initially here for breaking the testing. You are right. They can merge together.